src/components/Car.js

```diff
@@ -1,12 +1,34 @@
-import React from 'react'
-import cars from '../cars.json'
-// import material ui components here //
-// Container, Paper, Chip //
+import React from "react";
+import { Card, CardContent, CardActions, Divider } from "@material-ui/core";
+import cars from "../cars.json";
+import { Link } from "react-router-dom";
 
-const Car = (props) => {
-    return (
-        <h1>A specific car</h1>
-    )
-}
+const Home = () => {
+	console.log("CARS", cars);
+	return (
+		<div className="card-container">
+			{cars.map((car, idx) => (
```

Comment:
For this route (`/cars/:id` which is a URL like `/cars/1234`) the idea was to find that _one_ care and just show _it's_ details.  So the change you would make is to add a `find` before the `return` which would look through all the `cars` and find just the one that matches `props.match.params.id`.  When you use that special syntax with the colon (`/card/:id`) then `react-router` will capture it for you and put it in the `params` object which is in the `match` object pass in as a `prop`.  Something like this is what you were missing:

```
const car = cars.find( c => c.id === props.match.params.id );
```

then you would use `car` and show it's details instead of `map`ing.